grid_common/grid_protocol.h

```diff
@@ -393,6 +393,14 @@
 #define GRID_LUA_FNC_G_ELEMENT_COUNT_fnptr l_grid_element_count
 #define GRID_LUA_FNC_G_ELEMENT_COUNT_usage "element_count(void) Returns the number of elements on the current module."
 
+#define GRID_LUA_FNC_G_POTMETER_CALIBRATION_GET_short "gpcg"
+#define GRID_LUA_FNC_G_POTMETER_CALIBRATION_GET_human "potmeter_calibration_get"
+#define GRID_LUA_FNC_G_POTMETER_CALIBRATION_GET_fnptr l_grid_potmeter_calibration_get
```

Comment:
Always add usage explaining string (similar to #define GRID_LUA_FNC_G_ELEMENT_COUNT_usage) for use in editor UX

Reply:
will add in a commit after 9d39ceb